### PR TITLE
docs: create security.md #1242

### LINKED ISSUE #1242: add security policy
currently, we don't have any security policy, we need to add it.

file name - `SECURITY.md` 

### Content 
```
** Reporting Security Issues**

We want everyone who uses or helps with the project to tell us quickly if they find a problem that could make the project less safe. If you see something like this or think there's a problem related to safety, please do these things:

- Send an email to our team hello@reactplay.io, and tell us what the problem is in as much detail as you can.
- You can tell us on our [discord server](https://discord.gg/bGQqyM9g).
- In the subject line of your email, make it clear that it's about security. For example, you can write something like "A Problem with Safety - Project Name."
- Explain what the safety problem is in a simple and short way. If you have any proof, like pieces of code or logs, include them in your email.
- Don't talk about the problem openly or on the internet until we've had a chance to look at it. We want to fix it before it becomes a bigger issue.
```

### PR DESCRIPTION
**Pull request by alishabhale**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #1242 

## Type of change

- [x] This change requires a documentation update

# How Has This Been Tested?

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+** Reporting Security Issues**
```
Comment (priyankarpal) [inline SECURITY.md:1]: just need to change a little bit

``` suggestion
## Reporting Security Issues
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — COMMENTED** [review]
@alishabhale please add issue number to the PR description

**Comment by alishabhale** [comment]
> @alishabhale please add issue number to the PR description

@priyankarpal  Done. Please review.

**Comment by priyankarpal** [comment]
> > @alishabhale please add issue number to the PR description
> 
> @priyankarpal Done. Please review.

add issue number here (e.g. fix issue #1242 )

image

**Comment by alishabhale** [comment]
> > > @alishabhale please add issue number to the PR description
> > 
> > 
> > @priyankarpal Done. Please review.
> 
> add issue number here (e.g. fix issue #1242 )
> 
> image

@priyankarpal  Done

**Comment by priyankarpal** [comment]
@alishabhale you're changing only the PR title, that's good but make sure to add issue number to the PR description.
- also, for documentation-related changes, the PR title will be `docs: create security.md #1242 `
- you can check the Conventional Commits guide it's really useful  https://www.conventionalcommits.org/en/v1.0.0/ 
image

**Review by priyankarpal — CHANGES_REQUESTED** [review]
this discord invite link is not valid

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Review by atapas — APPROVED** [review]

**Comment by atapas** [comment]
Thanks @alishabhale for contributing.

**Pull request merged**